src/openrct2/ride/Ride.cpp

```diff
@@ -3551,14 +3552,18 @@ ResultWithMessage Ride::CreateVehicles(const CoordsXYE& element, bool isApplying
  * preceding that block.
  *  rct2: 0x006DDF9C
  */
-void Ride::MoveTrainsToBlockBrakes(TrackElement* firstBlock)
+void Ride::MoveTrainsToBlockBrakes(const CoordsXYZ& firstBlockPosition, TrackElement& firstBlock)
```

Comment:
firstBlockPosition is only used in speedy block brakes, it just got included because of interactive commit.